drivers/volume/common.go

```diff
@@ -957,3 +957,11 @@ func (d *DefaultDriver) GetRebalanceJobStatus(jobID string) (*api.SdkGetRebalanc
 		Operation: "GetRebalanceJobStatus()",
 	}
 }
+
+// UpdatePoolsLabels updates the labels of the desired pool, by appending a custom key-value pair
+func (d *DefaultDriver) UpdatePoolsLabels(n node.Node, poolID string, labels map[string]string) error {
```

Comment:
This function updates labels for a single given pool ID, please name it `UpdatePoolLabels` instead, since it doesn't update multiple pools

Reply:
Any reason you want to mentions that this functions updates multipole `Pools` and not a single `Pool`?